e2e/tests/core/03-connection/connection_test.go

```diff
@@ -0,0 +1,89 @@
+package connection
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	paramsproposaltypes "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	"github.com/strangelove-ventures/ibctest/ibc"
+	"github.com/strangelove-ventures/ibctest/test"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	"github.com/cosmos/ibc-go/e2e/testvalues"
+	transfertypes "github.com/cosmos/ibc-go/v5/modules/apps/transfer/types"
+	connectiontypes "github.com/cosmos/ibc-go/v5/modules/core/03-connection/types"
+	ibctesting "github.com/cosmos/ibc-go/v5/testing"
+)
+
+func TestConnectionTestSuite(t *testing.T) {
+	suite.Run(t, new(ConnectionTestSuite))
+}
+
+type ConnectionTestSuite struct {
+	testsuite.E2ETestSuite
+}
+
+// QueryConnectionEnabledParam queries the on-chain connection enabled param for 03-connection
+func (s *ConnectionTestSuite) QueryMaxExpectedTimePerBlockParam(ctx context.Context, chain ibc.Chain) uint64 {
+	queryClient := s.GetChainGRCPClients(chain).ParamsQueryClient
+	res, err := queryClient.Params(ctx, &paramsproposaltypes.QueryParamsRequest{
+		Subspace: "ibc",
+		Key:      string(connectiontypes.KeyMaxExpectedTimePerBlock),
+	})
+	s.Require().NoError(err)
+
+	// removing additional strings that are used for amino
+	delay := strings.ReplaceAll(res.Param.Value, "\"", "")
```

Comment:
I wasn't exactly sure why the Value is double wrapped in quotes but based on an error I received when I don't add them it is something to do with the legacy amino implementation. 

cc: @colin-axner

Reply:
That's a good question. I'm not sure either (didn't realized this was the case). I guess we can fix this during param migrations